packages/storybook-utils/src/actions.ts

```diff
@@ -77,9 +109,39 @@ export const defineActions = <T>(events: ExtractVueEventNames<T>[]): ArgTypes =>
  * }
  * ```
  */
-export const withVModelDecorator = <T>(events: ExtractVueEventNames<T>[]): Decorator => {
+export function withVModelDecorator<T>(events: ExtractVueEventNames<T>[]): Decorator;
+/**
+ * Defines a custom decorator that will implement event handlers for all v-models
+ * so that the Storybook controls are updated live when the user interacts with the component
+ *
+ * @example
+ * ```ts
+ * // .storybook/preview.ts
+ *
+ * {
+ *   decorators: [withVModelDecorator()]
+ * }
+ * ```
+ */
+export function withVModelDecorator(options?: WithVModelDecoratorOptions): Decorator;
```

Comment:
Shouldn't this log all events, not only v-models? Also click events etc. should be logged

Reply:
thats what `enhanceEventArgTypes` is for :)